ArticleService.js

```diff
@@ -0,0 +1,53 @@
+export function getArticleList(params = {}) {
+  const url = new URL("https://panda-market-api-crud.vercel.app/articles");
+  Object.keys(params).forEach((key) =>
+    url.searchParams.append(key, params[key])
+  );
+  return fetch(url)
+    .then((res) => res.json())
+    .then((data) => data);
```

Comment:
불필요한 체이닝은 제거하셔도 됩니다!

```suggestion
    .then((res) => res.json());
```